test/e2e/directory_permissions_test.go

```diff
@@ -0,0 +1,171 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/vmware-tanzu/carvel-vendir/pkg/vendir/config"
+	"gopkg.in/yaml.v2"
```

Comment:
We are using this library inside the code, maybe it makes sense to keep it consistent. I just realized that the gopkg one is used in a different tests, I will create a PR to fix that as well
```suggestion
	"sigs.k8s.io/yaml"
```